task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -1524,6 +1526,11 @@ def test_handle_requests(
         next(generator)
         msg = message.model_dump_json().encode() + b"\n"
         generator.send(msg)
+
+        if isinstance(message, GetVariable):
+            mock_mask_secret.assert_called_once()
+            mock_mask_secret.assert_called_with("test_value", "test_key")
```

Comment:
This has a change to be sliently never checked if we accidentally change the GetVariable fixture. It’s probably better to check these in a separate test.

Reply:
@uranusjr GetVariable is a class not a fixture from comms.py, if we bulk update it, this test gets covered too. Although i guess we can handle it more generally?